### PR TITLE
delete bug corrected / create add Component / add CRUD put

### PR DESCRIPTION
**Pull request by HenriBom**
… component

### CODE REVIEW

**Review by cocap10 — CHANGES_REQUESTED** [review]
you must not commit all files in node_modules dir

File: src/RecipeTable.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ class IsRecipe extends Component {
     if(this.props.recipe.length === 0) {
       return(<h2>there is no recipe :'(</h2>)
```
Comment (cocap10) [inline src/RecipeTable.js]: why `'` ?

File: src/RecipeTable.js [hunk 2]
```diff
@@ -27,7 +26,12 @@ class IsRecipe extends Component {
...
                 <td>{recipe.creator}</td>
                 <td>
-                  <button className="button smallButton">Edit</button>
+                  <button className="button smallButton" onClick={() => this.props.edit(
+                    recipe.name,
+                    recipe.ingredientType,
+                    recipe.malt,
+                    recipe.creator,
+                    recipe.id)}>Edit</button>
                   <button className="button smallButton" onClick={() => this.props.delete(recipe.name)}>Delete</button>
```
Comment (cocap10) [inline src/RecipeTable.js]: You should create a method in this classe to replace `() => this.props.delete(recipe.name)`

**Review by cocap10 — COMMENTED** [review]
IsRecipe must be in his own file

File: src/RecipeTable.js (hunk 1 above)
Comment (HenriBom) [reply src/RecipeTable.js]: Just to print a smiley :'(

File: src/RecipeTable.js (hunk 2 above)
Comment (HenriBom) [reply src/RecipeTable.js]: in ```() => this.props.delete(recipe.name)``` I am calling the ```Delete``` method from component Home. If I create a method for this matter wouldn't it do the same thing ?

**Review by cocap10 — APPROVED** [review]

**Pull request merged**